ts-orm/sequelize/model.ts

```diff
@@ -20,13 +20,13 @@ async function setupSequelize(): Promise<Sequelize> {
    *   id = "sequelizeDB"
    * }
    * */ 
-  const sequelize = new Sequelize(`sqlite::memory:`, { logging: false });
+  const sequelizez = new Sequelize(`sqlite::memory:`, { logging: false });
```

Comment:
might have been a fat finger extra letter z at the end

Reply:
yup let me fix